runtime/v1/linux/task.go

```diff
@@ -85,8 +85,8 @@ func (t *Task) Namespace() string {
 }
 
 // PID of the task
-func (t *Task) PID() uint32 {
-	return uint32(t.pid)
+func (t *Task) PID(_ctx context.Context) (uint32, error) {
```

Comment:
Typo? `_ctx`

Reply:
No, context is not used in v1 runtime, but needs to be passed in order to implement the interface. So it's marked with `_` prefix to be ignored by linters.

Reply:
Ah, right; I usually just use `_` (no variable name) for that

Reply:
look like rust style. go use `_`